pkg/kubelet/apis/podresources/server_v1_test.go

```diff
@@ -112,7 +112,12 @@ func TestListPodResourcesV1(t *testing.T) {
 			cpus:             []int64{},
 			memory:           []*podresourcesapi.ContainerMemory{},
 			dynamicResources: []*podresourcesapi.DynamicResource{},
-			expectedResponse: &podresourcesapi.ListPodResourcesResponse{},
+			expectedResponse: &podresourcesapi.ListPodResourcesResponse{
```

Comment:
We need to have some tests to test the backwards compatibility.

Reply:
I wonder how to do that but in this very case I can perhaps somehow reuse the v1alpha1 client or somehow ship the previous version of the proto/compiled bindings